README.md

```diff
@@ -69,6 +69,15 @@ if(Bro(object).doYouEven('lift') === Bro.TOTALLY) {
 }
 ```
 
+Or check for multiple nested members by passing an array of paths
```

Comment:
Might be more clear to say,

    Or, ensure that multiple nested members exist by passing an array of paths